src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.test.tsx

```diff
@@ -25,23 +24,18 @@ describe("KVMResourcesCard", () => {
     jest.restoreAllMocks();
   });
 
-  it("fetches machines on load", () => {
+  it("fetches machines on load", async () => {
```

Comment:
I don't think this needs to be async, there's no asynchronous calls in this test